tests/functional/test_issue-228/main.tf

```diff
@@ -0,0 +1,15 @@
+provider "aws" {
+  access_key = "foo"
+  secret_key = "bar"
+  skip_credentials_validation = true
+  skip_requesting_account_id = true
```

Comment:
This is not related with this PR, but as long as aws credentials in your environment ( e.g. if you use tools like [awsume](https://github.com/trek10inc/awsume) ), you don't need to have this 4 lines :) AWS SDK will detect it automatically from the environment variables.

Reply:
I don't really have credentials available all the time, this avoids any validation. It's a bit noisy but it works.